packages/run-protocol/src/reserve/params.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import {
+  CONTRACT_ELECTORATE,
+  makeParamManager as makeBaseParamManager,
+  ParamTypes,
+} from '@agoric/governance';
+
+export const AMM_INSTANCE_KEY = 'AmmInstance';
+
+/**
+ * @param {ERef<ZoeService>} zoe
+ * @param {Instance} ammInstance
+ * @param {Invitation} poserInvitation - invitation for the question poser
+ */
+const makeParamManager = async (zoe, ammInstance, poserInvitation) => {
+  return makeBaseParamManager(
```

Comment:
non-blocking: let `makeParamManager` be the general one
```suggestion
const makeReserveParamManager = async (zoe, ammInstance, poserInvitation) => {
  return makeParamManager(
```

Either way please make the exports either both have `Reserve` or neither.

Reply:
Done